docs/ref/models/querysets.txt

```diff
@@ -1991,6 +1991,15 @@ from the database without loading the entire result set into memory.
     PostgreSQL support for server-side cursors was added.
 
 
+.. versionadded:: 1.11
+
+``chunk_size`` is used to specify the number of results to cache at the
```

Comment:
Move this above the .". versionchanged::" (the annotations should be 'self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features